src/styles/mixins.scss

```diff
@@ -0,0 +1,50 @@
+@mixin input-element {
```

Comment:
[nice to have / maybe on another branch]
maybe it is time to refactor input components and create a generic `MainInput`? 🤔

You know the tools (e.g. props, slots). You don't have to abandon having specific inputs like `PasswordInput`, but you can make them use this generic component inside. I very much like the idea of reusing components instead of reusing styles - it's easier to manage, easier to test and easier introduce changes.

Reply:
I will start a new branch and get this done 😄 
I completelly agree with what you are saying - It was always on to do list, just not got round to it yet 😄